src/odemis/gui/cont/stream.py

```diff
@@ -198,6 +198,12 @@ def __init__(self, stream_bar, stream, tab_data_model, show_panel=True, view=Non
             self.stream.zIndex.subscribe(self._on_z_index)
             self.tab_data_model.zPos.subscribe(self._on_z_pos, init=True)
 
+        if hasattr(stream, "zIndex") and hasattr(stream, "max_projection"):
+            for se in self.entries:
+                if se.vigilattr == self.stream.zIndex:
+                    self._zindex_se = se
```

Comment:
Comparison to find an instance should be just a `is` (it's faster).
Also, it's good to handle the unexpected case where it fails to find the zIndex.
```suggestion
            for se in self.entries:
                if se.vigilattr is self.stream.zIndex:
                    self._zindex_se = se
                    break
            else:
                 logging.warning("Stream has zIndex but no corresponding stream entry found")
```